sort.php

```diff
@@ -20,12 +20,27 @@ function bubbleSort($sort_array)
 
     for ($i = 0; $i < $array_indexes; $i++) {
         for ($j = 0; $j < $array_indexes - $i - 1; $j++) {
-            if ($sort_array[$j + 1] < $sort_array[$j]) {
-                $tmp = $sort_array[$j + 1];
-                $sort_array[$j + 1] = $sort_array[$j];
-                $sort_array[$j] = $tmp;
+            if (getopt("n")) {
```

Comment:
Call getopt() only once. For whether a parameter is given consult the return value of that one call.

Reply:
You're still calling getopt() more than once in the whole program.

Reply:
Should I use the global key word?

Reply:
No, you should add a parameter to bubbleSort().